webdriver-spec.html

```diff
@@ -4010,7 +4019,43 @@
 </section> <!-- /Element Interaction -->
 
 <section>
-<h2>Executing Script</h2>
+<h2>Document Handling</h2>
+<section>
+<h3>Getting Page Source</h3>
+
+<table class="simple jsoncommand">
+ <tr>
+  <th>HTTP Method</th>
+  <th>Path Template</th>
+ </tr>
+ <tr>
+  <td>GET</td>
+  <td>/session/{<var>session id</var>}/source</td>
+ </tr>
+</table>
+
+<p>The <dfn>Get Page source</dfn> <a>command</a> returns the HTML source of the
+  document as it is at the time of the call.
```

Comment:
True, I was just thinking about the amount of users who I have helped understand what Page Source returns since its not the HTML that is initially sent over the wire. They forget that their JS could be mutating the DOM.

Updated as suggested